### PR TITLE
SyncEngine: invalid the blacklist entry when the rename destination c…

### PR DESCRIPTION
**Pull request by ogoffart**
…hange

IL issue 558

### CODE REVIEW

File: src/libsync/syncengine.cpp [hunk 1]
```diff
@@ -214,6 +214,9 @@ bool SyncEngine::checkErrorBlacklisting( SyncFileItem &item )
         } else if( item._modtime != entry._lastTryModtime ) {
             qDebug() << item._file << " is blacklisted, but has changed mtime!";
             return false;
+        } else if( item._renameTarget != entry._renameTarget) {
+            qDebug() << item._file << " is blacklisted, byt renameTarget changed from" << entry._renameTarget;
```
Comment (guruz) [inline src/libsync/syncengine.cpp]: byt?

Comment (guruz) [inline src/libsync/syncengine.cpp]: Could you also add a comment explaining this? Why is it not blacklisted then? for future...

Comment (ogoffart) [inline src/libsync/syncengine.cpp]: that's what every other branch in this function does.  I don't think one need a comment for that in this branch and not the others.

**Comment by guruz** [comment]
👍 

Also needs to be put into 2.2

**Comment by guruz** [comment]
So you had merged 2.1 in 2.2/master after all? So this is fine now in master?

**Comment by ogoffart** [comment]
yes, it's in 2.2

**Pull request merged**